common/djangoapps/philu_commands/management/commands/tests/test_auto_generate_certificates_for_open_courses.py

```diff
@@ -0,0 +1,133 @@
+        mock_modulestore.return_value = mock_response
+        mock_objects = CourseEnrollmentFactory.create(
+            user=self.user, course_id=self.course_1.id, mode='honor')
+        mock_course_enrollments.return_value = Mock(**{"all.return_value": [mock_objects], "get_items": {}})
+        mock__get_cert_data.return_value = Mock(**{"cert_status": "requesting"})
+
+        def assert_certificates_generated(info):
+            """
+            assert certification generation
+            """
+            certificate_success_message = 'Generating certificate for user with ' \
+                                          'username: {} and user_id: {} with ' \
+                                          'generation status: {}'.format(self.user.username, self.user.id,
+                                                                         mock_generate_user_certificates())
+            course_id_message = 'course id : {course_id}'.format(course_id=self.course_1.id)
+            assert info in (certificate_success_message, course_id_message)
+
+        mock_log_info.side_effect = assert_certificates_generated
+        call_command('auto_generate_certificates_for_open_courses')
+        mock_log_info.assert_called(2)
+
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.modulestore')
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.log.info')
+    def test_no_open_course_available(self, mock_log_info, mock_modulestore):
+        """
+        Test 'If no open course is available'
+        """
+        mock_response = Mock(**{"get_courses.return_value": [self.course_1]})
+        mock_modulestore.return_value = mock_response
+        call_command('auto_generate_certificates_for_open_courses')
+        assert not mock_log_info.called
+
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.modulestore')
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.log.info')
+    def test_course_not_valid_for_certificate_auto_generation(self, mock_log_info, mock_modulestore):
+        """
+        Test 'If course is not valid'
+        """
+        mock_response = Mock(**{"get_courses.return_value": [self.course_1]})
+        mock_modulestore.return_value = mock_response
+        call_command('auto_generate_certificates_for_open_courses')
+        assert not mock_log_info.called
+
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.modulestore')
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.log.info')
+    @patch('philu_commands.management.commands.auto_generate_certificates_for_open_courses.'
+           'is_course_valid_for_certificate_auto_generation')
+    def test_course_valid_cert_data_not_exists(self, mock_is_course_valid, mock_log_info,
+                                               mock_modulestore):
```

Comment:
No need for line break. If you want to break the line, Maybe the better way is 
```suggestion
def test_course_valid_cert_data_not_exists(
    self, mock_is_course_valid, mock_log_info, mock_modulestore):
```